x-pack/plugins/monitoring/server/plugin.ts

```diff
@@ -267,6 +271,8 @@ export class MonitoringPlugin
   }
 
   registerPluginInUI(plugins: PluginsSetup) {
+    const alertingRules = [...RULES, TRANSFORM_RULE_TYPE.TRANSFORM_HEALTH];
```

Comment:
Is there a reason we're not adding `TRANSFORM_RULE_TYPE.TRANSFORM_HEALTH` to the `RULES` constant directly?

Reply:
I tried it at first but `RULES` are used by the factory [here](https://github.com/elastic/kibana/blob/bc25c0fca95ace715d8ed026941ae8bd68ccf056/x-pack/plugins/monitoring/server/alerts/alerts_factory.ts#L84). So it requires extending the `BaseRule` class which I assume is redundant for the Transform alerting rule.